ethereum/api/src/main/java/org/hyperledger/besu/ethereum/api/jsonrpc/JsonRpcHttpService.java

```diff
@@ -165,12 +167,7 @@ private void validateConfig(final JsonRpcConfiguration config) {
     LOG.info("Starting JsonRPC service on {}:{}", config.getHost(), config.getPort());
 
     // Create the HTTP server and a router object.
-    httpServer =
-        vertx.createHttpServer(
-            new HttpServerOptions()
-                .setHost(config.getHost())
-                .setPort(config.getPort())
-                .setHandle100ContinueAutomatically(true));
+    httpServer = vertx.createHttpServer(getHttpServerOptions());
```

Comment:
May I offer to use this approach? 

https://github.com/apache/incubator-tuweni/blob/master/net/src/test/java/org/apache/tuweni/net/tls/ServerRecordTest.java#L120

Reply:
@atoulme The requirements for client authentication at the moment is that all clients are known in advance (primarily EthSigner). Tuweni's VertX trust store implementation is useful when we want to trust client after first connect.

Reply:
That's one option, you can also use whitelist.

Reply:
👍 . whitelist will work. I'll add that bit in for simplifying trust store requirements. @rain-on FYI.

Reply:
@atoulme If I only want to trust CA certificate (instead of self-signed client certificates), does tuweni net API provide any TrustOption for that?

Reply:
is -Djavax.net.ssl.trustStore is the standard approach to specify custom CA certificates instead of setting it up directly in vertx client trust options?